atom/browser/browser.cc

```diff
@@ -109,11 +112,23 @@ void Browser::Activate(bool has_visible_windows) {
 }
 
 void Browser::WillFinishLaunching() {
+  base::FilePath userDir;
+  PathService::Get(brightray::DIR_USER_DATA, &userDir);
```

Comment:
I'm not super excited to put this code here, but when you put this code in the Browser ctor, DIR_USER_DATA isn't set yet